core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3859,6 +3898,9 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 else if (wasm_is_type_reftype(local_type)) {
                     PUT_REF_TO_ADDR((uint32 *)(frame_lp + local_offset),
                                     GET_REF_FROM_ADDR(frame_lp + addr1));
+                    if (opcode == WASM_OP_SET_LOCAL) {
```

Comment:
This will cause a compilation warning `warning: ‘opcode’ may be used uninitialized in this function [-Wmaybe-uninitialized]`

This is very strange

Reply:
fast interpreter doesn't dispatch based on opcode, so how should we distinguish the local.set and local.tee?